west.yml

```diff
@@ -166,10 +166,10 @@ manifest:
       remote: nordicsemi
       revision: 1c2656daa87ebf4cd3c45ee8eafa336260d589bc
       path: modules/lib/cddl-gen
-    - name: memfault-firmware-sdk
-      path: modules/lib/memfault-firmware-sdk
-      revision: 0.27.3
-      remote: memfault
+#    - name: memfault-firmware-sdk
+#      path: modules/lib/memfault-firmware-sdk
+#      revision: 0.27.3
+#      remote: memfault
```

Comment:
We'll need a new tag/release for this, @jtguggedal

Reply:
Alright, I'll look into that

Reply:
I'm not sure we need a new release, it should be sufficient to remove this line from what I can see:
https://github.com/nrfconnect/sdk-nrf/blob/31b79a4d5b0160973fea544e80398c8d09db0919/modules/memfault-firmware-sdk/Kconfig#L100

We've reached out to Memfault and notified them of the removal of at_cmd and at_notif anyway so that their samples can be updated.